src/inmanta/data/dataview.py

```diff
@@ -357,10 +360,24 @@ async def _get_page_count(self, bounds: Union[PagingBoundaries, RequestedPagingB
                 page_size=self.limit,
             )
 
+        def construct_filter(filter_name: str, filter_condition: str, drop_condition: PagingOrder) -> str:
+            """
+            Construct filter to count the number of items that are before / after the current page.
+            The filtering might be dropped if no result is found and iff the order provided in drop_condition is the one used
+            in the current page.
+
+            :param filter_name: The name of the variable to store the result of the select
+            :param filter_condition: The condition to use in the filtering
```

Comment:
I'm not really happy with this part of the documentation: I'm not really inspired to explain simply when the filter part might be dropped. Any idea?